specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2023-06-01/managedClusters.json

```diff
@@ -6239,9 +6239,25 @@
       "description": "VPA (Vertical Pod Autoscaler) settings for the workload auto-scaler profile.",
       "properties": {
         "enabled": {
-          "type": "boolean",
+          "type": "string",
+          "enum": [
+            "true",
+            "false"
+          ],
+          "x-ms-enum": {
```

Comment:
IMO there is no value in doing this in an enum. Just tell the Swagger reviewer we have a pattern of using boolean for addon enablement and they will let us through.